docs/api/proposed_endpoints.md

```diff
@@ -0,0 +1,153 @@
+[TOC]
+
+# Posts related actions
+
+## Create a post
+
+Create a post of contents (text + image).
+
+```
+POST api/posts/create
```

Comment:
No need to give actions on `/api/posts/create`. 

In REST, `POST` already has the significance that we're creating a resource.